src/components/team/billing-section.tsx

```diff
@@ -72,14 +75,31 @@ const BillingSection = ({
   const activeSubscription =
     get(subscriptionData, 'subscription.status') === 'active' &&
     !get(subscriptionData, 'subscription.canceled_at')
-  const nextBillDate: string | undefined = activeSubscription
-    ? format(
-        new Date(
-          get(subscriptionData, 'upcomingInvoice.next_payment_attempt') * 1000,
-        ),
-        'yyyy/MM/dd',
-      )
-    : undefined
+
+  // Figure out what to display for the Next Billing Date.
```

Comment:
The `current_period_*` props on the `Subscription` object are likely to give a good indication https://stripe.com/docs/api/subscriptions/object